lib/erb_lint/linters/final_newline.rb

```diff
@@ -10,22 +10,22 @@ def initialize(config)
         @new_lines_should_be_present = config['present'].nil? ? true : config['present']
       end
 
-      protected
-
-      def lint_lines(lines)
+      def lint_file(file_tree)
```

Comment:
Is there no access to the source of the file? You're jumping through a lot of hoops to make this work with the parsed tree.